cmd/agent/subcommands/secret/command.go

```diff
@@ -61,49 +58,22 @@ func Commands(globalParams *command.GlobalParams) []*cobra.Command {
 }
 
 func showSecretInfo(config config.Component) error {
-	r, err := callAPIEndpoint("/agent/secrets", config)
-	if err != nil {
-		return err
-	}
-	fmt.Println(string(r))
-	return nil
+	return getAndPrintIPCEndpoint(config, "agent/secrets")
 }
 
 func secretRefresh(config config.Component) error {
-	r, err := callAPIEndpoint("/agent/secret/refresh", config)
-	if err != nil {
-		return err
-	}
-	fmt.Println(string(r))
-	return nil
+	return getAndPrintIPCEndpoint(config, "agent/secret/refresh")
```

Comment:
nit: Maybe splitting the printing part from getting the response would reduce the amount of things the function does 😄